docs/source/configuration.rst

```diff
@@ -103,15 +103,17 @@ Defaults to ``https://s3.amazonaws.com/<aws_region_name>-cfncluster/templates/cf
 
 compute_instance_type
 """""""""""""""""""""
-The EC2 instance type used for the cluster compute nodes.
+The EC2 instance type used for the cluster compute nodes. Note that when using a placement group
+only certain instance types can participate in a placement group.
```

Comment:
More concise: "Note that when using a placement group
only certain instance types may be used, see :ref:`Placement Group <placement_group>`.."